binrz/man/rz-ax.1

```diff
@@ -1,100 +1,100 @@
-.Dd Dec 28, 2020
+.Dd Jan 24, 2024
 .Dt RZ_AX 1
 .Sh NAME
 .Nm rz-ax
 .Nd rizin base converter
 .Sh SYNOPSIS
 .Nm rz-ax
-.Op Fl ebBsSvxkKh
+.Op Fl abBdefFhiIkLlNnorsStuvwxC
 .Op [expr] ...
 .Sh DESCRIPTION
 This command is part of the rizin project.
 .Pp
-This command allows you to convert values between positive and negative integer, float, octal, binary and hexadecimal values.
+This command allows you to convert values between positive and negative integers, floats, octal, binary, ternary, and hexadecimal values.
 .Sh OPTIONS
 .Bl -tag -width Fl
+.It Fl l
+Append newline to output (for -E/-D/-r/..)
```

Comment:
There is no `-E` nor `-D` option.

Reply:
I'm trying `-l` option right now and i'm not sure i understand what it does :/ we should find out and update the doc here.